Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -0,0 +1,271 @@
+﻿function Publish-NewServiceFabricApplication
```

Comment:
I'm guessing these SDK files are largely copy/paste from a different repo?

Fyi - This reminds me, if you find yourself needing to share common code between different Service Fabric tasks, you can put it under `/Tasks/Common` and add an entry under `/common.json`. It will copy the common code into place at build time. For example, there is common code shared between the MSBuild task and VSBuild task.